scripts/storage/secondary/createvolume.sh

```diff
@@ -135,7 +135,8 @@ create_from_file() {
   local tmpltname=$3
 
   [ -n "$verbose" ] && echo "Moving to $tmpltfs/$tmpltname...could take a while" >&2
-  mv $tmpltimg /$tmpltfs/$tmpltname
+  cp -rf $tmpltimg /$tmpltfs/$tmpltname
```

Comment:
mv maybe faster?

Reply:
@rhtyd y, mv is faster, however for configdrive ISO, if it is attached to a vm, it will not be recognized in vm after "mv"
there is no issue if "cp" is used.

I am thinking about (1) not change this script, but copy configdrive ISO in java instead of calling this script (createvolume.sh); (2) detach and re-attach configdrive ISO when it is changed.

this is a draft PR, still WIP.